frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/CheckoutSelectedBranchAction.java

```diff
@@ -92,6 +92,7 @@ public void run(ProgressIndicator indicator) {
   @UIThreadUnsafe
   public static void doCheckout(Project project, ProgressIndicator indicator, String branchToCheckoutName,
       GitRepository gitRepository) {
+    // TODO (#772): switch to constructor that does not take git once we no longer support 2021.2
```

Comment:
The version with `git` param is deprecated?

Reply:
yup